lib/ansible/modules/unarchive.py

```diff
@@ -141,6 +141,66 @@
     - s/^xxx/yyy/
 '''
 
+RETURN = r'''
+dest:
+  description: Path to the destination directory
+  returned: always
+  type: str
+  sample: /opt/software
+files:
+  description: List of all the file in the archive.
+  returned: When I(list_files) is True
+  type: list
+  sample: '["file1", "file2"]'
+gid:
+  description: numerical id of the group that owns the destination directory.
+  returned: always
+  type: int
+  sample: 1000
+group:
+  description: name of the group that owns the destination directory.
+  returned: always
+  type: str
+  sample: "paul"
```

Comment:
I'd prefer a plural noun as the example group name, especially since it appears in the documentation before the user name. Maybe if the user is "paul" the group could be "librarians" or something?